paper-toast.html

```diff
@@ -43,16 +43,24 @@
         margin: 12px;
         font-size: 14px;
         cursor: default;
-        -webkit-transition: visibility 0.3s, -webkit-transform 0.3s, opacity 0.3s;
-        transition: visibility 0.3s, transform 0.3s, opacity 0.3s;
+        -webkit-transition: -webkit-transform 0.3s, opacity 0.3s;
+        transition: transform 0.3s, opacity 0.3s;
 
-        visibility: hidden;
```

Comment:
animating and using `visibility` is not really needed, as we already animate `opacity`, and `IronOverlayBehavior` will set `display: none` when the overlay is closed